src/app/cad-board/cad-board.component.ts

```diff
@@ -3,30 +3,31 @@ import {
   OnInit,
   ViewChild,
   OnDestroy,
-}                         from '@angular/core'
+} from '@angular/core'
 import { Subscription } from 'rxjs'
 
 import {
   GoogleChartComponent,
-}                         from 'angular-google-charts'
+} from 'angular-google-charts'
 
 import {
   GpuService,
-}               from '../gpu.service'
+} from '../gpu.service'
+import 'element-angular/theme/index.css'
 
 interface GoogleChart {
-  title       : string,
-  type        : string,
-  data        : Array<Array<string | number | {}>>,
-  roles       : Array<{type: string, role: string}>,
+  title: string,
+  type: string,
+  data: Array<Array<string | number | {}>>,
+  roles: Array<{ type: string, role: string }>,
```

Comment:
![image](https://user-images.githubusercontent.com/1361891/48685520-aef90080-ebf1-11e8-99ed-c89141d4d08f.png)

Please keep the original format: vertical align the `:`, thanks.